asserts/model.go

```diff
@@ -722,6 +716,15 @@ func assembleModel(assert assertionBase) (Assertion, error) {
 		return nil, err
 	}
 
+	// distribution is optional and only for classic
+	distro, err := checkOptionalString(assert.headers, "distribution")
```

Comment:
We probably want to limit the distribution to only allow `[a-zA-Z]` initially. This way we can be sure that if we extend the syntax later to include things like "ubuntu > 20.04 or fedora = 36 will work and that the model is not already having ">=" and simialr and also that people did not write "distro: fedora-36.0" or similar already. But probably best to get @pedronis input how strict this regexp should (but I'm pretty sure we will need one and the matching tests).

Reply:
this can only appear if extended is true and classic true, also in that case is not optional

Reply:
yes, it would be good to have a regexp, is there a spec for what is accepted in os-release ID ?

Reply:
Done now - yes, there is actually a spec: https://www.freedesktop.org/software/systemd/man/os-release.html#ID=